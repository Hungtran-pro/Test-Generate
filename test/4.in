37
96
83
109
93
106
87
103
61
76
63
91
86
69
75
67
99
120
108
117
106
84
64
106
89
108
110
64
82
111
68
93
86
105
96
103
81
80
