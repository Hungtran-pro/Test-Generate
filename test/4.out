103131504
677424793
612955947
482361088
56249491
241802082
981043477
312450027
376671503
881779447
672789790
980141720
552589473
255840229
718461253
623131616
951234175
419333973
273207878
56249491
130003129
810370359
56249491
823798411
419333973
170399885
810370359
639909956
466486836
599807473
482361088
980141720
676689719
103131504
981043477
459405516
991943513
